app/scripts/modules/core/src/managed/ObjectRow.tsx

```diff
@@ -7,17 +7,18 @@ import styles from './ObjectRow.module.css';
 interface IObjectRowProps {
   icon: IconNames;
   title: string;
+  link?: string;
   metadata?: JSX.Element;
   depth?: number;
 }
 
-export const ObjectRow = ({ icon, title, metadata, depth = 1 }: IObjectRowProps) => {
+export const ObjectRow = ({ icon, title, link, metadata, depth = 1 }: IObjectRowProps) => {
   return (
     <div className={styles.ObjectRow} style={getStylesFromDepth(depth)}>
       <span className="clickableArea">
         <div className={styles.leftCol}>
           <Icon name={icon} size="medium" appearance="dark" className="sp-margin-s-right" />
-          <span className={styles.rowTitle}>{title}</span>
+          <span className={styles.rowTitle}>{link ? <a href={link}>{title}</a> : title}</span>
```

Comment:
this `link` prop is a little bit brittle in the sense that it's directly tied to the usage pattern we currently have (wrapping the title in an `<a>` tag for which we already have a fully constructed href). Instead, can we change the type `title` to `JSX.Element | string` and pass in the components/markup we want from _outside_ of the `ObjectRow` code?

Reply:
Good call. I'll update that